docs/core/components/dynamic-chip-list.component.md

```diff
@@ -31,9 +32,11 @@ This component shows dynamic list of chips which render depending on free space.
 |---------------------|---------------------------------------------------------------------------------|---------------|---------------------------------------------|
 | limitChipsDisplayed | `boolean`                                                                       | false         | Should limit number of chips displayed.     |
 | showDelete          | `boolean`                                                                       | true          | Show delete button.                         |
+| roundUpChips        | `boolean`                                                                       | false         | Round up chips.                             |
```

Comment:
Maybe something more descriptive here? For example `increasing the border radius of a chip to 20px` otherwise it's hard to know what the expected behaviour should be

Reply:
added :)